lib/models/food.js

```diff
@@ -20,6 +20,39 @@ function create(req) {
     });
   })
 }
+function find (id) {
+  return database.raw('SELECT * FROM foods WHERE id = ?', id)
+};
+
+function update(req) {
+  return new Promise( (resolve, reject) => {
+    const food = req.body.food
+
+    let query;
+
+    if (!food.calories && food.name) {
```

Comment:
There are features in knex that would help us handle the updating, but since we're using raw SQL, I think the only way to handle it is this conditional.